lib/dates.rb

```diff
@@ -0,0 +1,38 @@
+require 'date'
+require 'pry'
+
+class Dates
+  attr_reader :date, :date_range
```

Comment:
You have `attr_reader`s here, but I don't think you ever make instances of this class.